oauth2/oauth2-resource-server/src/test/java/org/springframework/security/config/annotation/web/configurers/oauth2/resourceserver/ResourceServerConfigurerTests.java

```diff
@@ -133,12 +133,12 @@ protected void configure(HttpSecurity http) throws Exception {
 
 			http
 					.authorizeRequests().anyRequest()
-					.access("authentication.hasClaim('scope', 'permission.read')");
+					.access("authentication.hasAttribute('scope', 'permission.read')");
```

Comment:
If you take a look at my open pull request, you'll see that these changes are in place there as well. I haven't taken a look at it in a while, but we'll probably need to reconcile the two.

Reply:
For sure I will go back to the PR and do the manual merge